pkg/cluster/cluster.go

```diff
@@ -80,16 +80,16 @@ type manager struct {
 	subnet  subnet.Manager
 	graph   graph.Manager
 
-	kubernetescli  kubernetes.Interface
-	extensionscli  extensionsclient.Interface
-	maocli         maoclient.Interface
-	mcocli         mcoclient.Interface
-	operatorcli    operatorclient.Interface
-	configcli      configclient.Interface
-	samplescli     samplesclient.Interface
-	securitycli    securityclient.Interface
-	arocli         aroclient.Interface
-	registryclient imageregistryclient.Interface
+	kubernetescli       kubernetes.Interface
+	extensionscli       extensionsclient.Interface
+	maocli              maoclient.Interface
+	mcocli              mcoclient.Interface
+	operatorcli         operatorclient.Interface
+	configcli           configclient.Interface
+	samplescli          samplesclient.Interface
+	securitycli         securityclient.Interface
+	arocli              aroclient.Interface
+	imageregistryclient imageregistryclient.Interface
```

Comment:
nit: all the others are suffixed with `cli` can `imageregistryclient` be named the same?

Reply:
I agree with CLI being confusing and having them named client makes more sense. I don't want to hold up this pr on a small naming change

Reply:
Ok, I moved to `cli` naming to continue this standard since we don't feel that we should change that.